build-support/bin/deploy_to_s3.py

```diff
@@ -80,13 +80,16 @@ def deploy(scope: str | None = None) -> None:
 
     local_path = "dist/deploy"
     s3_dest = "s3://binaries.pantsbuild.org"
+    s3_dest_region = "us-east-1"
```

Comment:
Ok, makes sense. Thanks for digging. What does not make sense is that I do not, to my knowledge, have a region configured in my WSL Ubuntu distro where I successfully run the backfill script either.

Reply:
I know, nor do the wheel build shards, so I'm still digging into this